src/devices/sound/flt_biquad.h

```diff
@@ -0,0 +1,80 @@
+// license:BSD-3-Clause
+// copyright-holders:K.Wilkins,Couriersud,Derrick Renaud,Frank Palazzolo,Jonathan Gevaryahu
+#ifndef MAME_SOUND_FLT_BIQUAD_H
+#define MAME_SOUND_FLT_BIQUAD_H
+
+#pragma once
+
+//**************************************************************************
+//  TYPE DEFINITIONS
+//**************************************************************************
+
+// ======================> filter_biquad_device
+
+class filter_biquad_device : public device_t, public device_sound_interface
+{
+public:
+	enum
+	{
+		LOWPASS1P = 0,
+		HIGHPASS1P,
+		LOWPASS,
+		HIGHPASS,
+		BANDPASS,
+		NOTCH,
+		PEAK,
+		LOWSHELF,
+		HIGHSHELF
+	};
```

Comment:
Consider making this an `enum class` so that the compiler will warn you about playing fast and loose with conversions.

Reply:
This is now implemented.